gameflow.md

```diff
@@ -0,0 +1,64 @@
+# Gameflow
+
+## Game Logic
+
+The selected level starts and presents the user a random syllable (Hiragna / Katakana) as well as four possible answers.
+The available syllables depend on the selected level.
+
+If the correct answer was selected, the game immediately goes to the next syllable.
+
+If a wrong answer was selected, the game shows a quick indicator that something was wrong, removes a heart container and then goes to the next syllable.
+
+If the required amount of syllables have been answered, the game navigates to the success screen and informs the player how many Gems / Jewels he won as well as if he unlocked new things like syllables.
+
+If the user runs out of Heart Containers the game navigates to the failure screen.
+
+## Game Segments
+
+**Syllable**
+
+A single piece of gameplay in which the player has to select the correct counterpart for a given syllable in latin character.
+
+**Level**
+
+A collection of syllables the user has to complete. At the end of the level the user will be awarded in the form of gems.
```

Comment:
Note my point above concerning a rough estimate on how many Syllables we talk about here.